elife-1234567890-v2/elife-1234567890-v2.xml

```diff
@@ -21,6 +21,8 @@
             <article-id pub-id-type="doi">10.7554/eLife.1234567890</article-id>
             <!-- Specific version DOI -->
             <article-id pub-id-type="doi" specific-use="version">10.7554/eLife.1234567890.4</article-id>
+            <!-- Article version used to distinguish reviewed preprints from versions of record. For RPs, value would be "reviewed preprint" -->
+            <article-version article-version-type="publication-state">version of record</article-version>
```

Comment:
I agree `publication-state` is a good (better) descriptor, but given we're not using a specific vocabulary should we use the suggested term in JATS documentation, `status`, in order to attempt to be consistent with others maybe using that suggested usage?